share/ruby-build/truffleruby+graalvm-dev

```diff
@@ -1,19 +1,26 @@
 platform="$(uname -s)-$(uname -m)"
 case $platform in
 Linux-x86_64)
-  install_package "truffleruby+graalvm-dev" "https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest/download/truffleruby-community-jvm-dev-linux-amd64.tar.gz" truffleruby
+  url="https://raw.githubusercontent.com/graalvm/graal-languages-ea-builds/main/truffleruby/versions/latest-jvm-linux-amd64.url"
   ;;
 Linux-aarch64)
-  install_package "truffleruby+graalvm-dev" "https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest/download/truffleruby-community-jvm-dev-linux-aarch64.tar.gz" truffleruby
+  url="https://raw.githubusercontent.com/graalvm/graal-languages-ea-builds/main/truffleruby/versions/latest-jvm-linux-aarch64.url"
   ;;
 Darwin-x86_64)
-  install_package "truffleruby+graalvm-dev" "https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest/download/truffleruby-community-jvm-dev-macos-amd64.tar.gz" truffleruby
+  url="https://raw.githubusercontent.com/graalvm/graal-languages-ea-builds/main/truffleruby/versions/latest-jvm-darwin-amd64.url"
   ;;
 Darwin-arm64)
-  install_package "truffleruby+graalvm-dev" "https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest/download/truffleruby-community-jvm-dev-macos-aarch64.tar.gz" truffleruby
+  url="https://raw.githubusercontent.com/graalvm/graal-languages-ea-builds/main/truffleruby/versions/latest-jvm-darwin-aarch64.url"
   ;;
 *)
   colorize 1 "Unsupported platform: $platform"
   return 1
   ;;
 esac
+
+pushd "$BUILD_PATH" >/dev/null
+http get $url url.txt
+url=$(<url.txt)
+popd
```

Comment:
The `pushd` is needed because `aria2c -o /absolute/path` doesn't work, it actually writes in `/cwd//absolute/path` which is the same as `/cwd/absolute/path`.
It would be tempting to just use `curl` to avoid the extra file, but that doesn't work on systems without `curl`.

Reply:
Yeah, the ruby-build script only ever calls `http` with a path-less filename so it just downloads to the CWD and eventually removes the file.
I worked around this once by just using a tempfile: https://github.com/Shopify/ruby-definitions/pull/36